android/capacitor/src/main/java/com/getcapacitor/cordova/CapacitorCordovaCookieManager.java

```diff
@@ -11,7 +11,7 @@
     public CapacitorCordovaCookieManager(WebView webview) {
         webView = webview;
         cookieManager = CookieManager.getInstance();
-        cookieManager.setAcceptFileSchemeCookies(true);
+        CookieManager.setAcceptFileSchemeCookies(true);
```

Comment:
@smpeters do you recall what the specific complaint was from Android Studio was regarding this? At first glance, the original looks correct.

Reply:
The complaint is that the method is a static method so it should be called using the class and not an instance.